package.json

```diff
@@ -124,7 +125,9 @@
     "mocha": "^5.0.0",
     "next": "^7.0.0",
     "nyc": "^13.0.0",
+    "patch-package": "^5.1.1",
```

Comment:
hummmmmmm, great for the speed of iteration and getting things done, but scarying for long terme support.

Reply:
Agreed. It also adds another source of potential confusing when reasoing about code origin (node module resolution, babel alias, webpack alias and now patch-package). 

Maybe we fork react-docgen for now under mui-org, pull it in as a git submodule in `packages/` and then build it on install.

Reply:
Can the changes be integrated into react-docgen?

Reply:
The `resolver`s and `handler`s for `react-docgen` are configurable. But if we need to yank most of their implementation and then pass it back in we might as well create a local fork.

The change itself is already proposed in reactjs/react-docgen#311 but the last commit was 26 days ago an pr/issue response time is in general pretty low on that repo. So I don't have high hopes that the PR will get much attention in the next month.

Reply:
Resolved due to release of `react-docgen@3.0.0`